airflow/models/log.py

```diff
@@ -73,7 +73,7 @@ def __init__(
         if task_instance:
             self.dag_id = task_instance.dag_id
             self.task_id = task_instance.task_id
-            if execution_date := getattr(task_instance, "execution_date", None):
+            if execution_date := getattr(task_instance, "logical_date", None):
                 self.execution_date = execution_date
```

Comment:
Do we need to change it as logical_date?

Reply:
Yes because in log we still use execution_date in this PR